internal/provider/resource_docker_image.go

```diff
@@ -22,7 +22,14 @@ func resourceDockerImage() *schema.Resource {
 
 			"latest": {
 				Type:        schema.TypeString,
-				Description: "The ID of the image.",
+				Description: "The ID of the image in the form of `sha256:<hash>` image digest. Do not confuse it with the default `latest` tag.",
```

Comment:
It looks a breaking change.

Reply:
It's not. I still use the `ID` like before. Just update the description, which is misleading. Although the `ID` has the same form as the digest, it's simply an `ID`

Reply:
I see. Thank you for your explanation.